docs/source/notebooks/Tutorial.md

```diff
@@ -0,0 +1,179 @@
+
+action = env.action_space.sample()
+
+data = env.step(action=action, return_state=False)
+observ, reward, end, info = data
+```
+
+By setting `return_state=False`, neither `reset()` nor `step()` will return the state of the simulation. In this way,
+we are obtaining the exact same answers as if we were working in a plain `gym` interface. Thus, `plangym`
+provides a complete tool for developing planning projects __as well as__ a general, standard API for reinforcement learning problems.  
+
+### Stepping a batch of states and actions
+```python
+import plangym
+env = plangym.make(name="CartPole-v0")
+state, obs = env.reset()
+
+states = [state.copy() for _ in range(10)]
+actions = [env.action_space.sample() for _ in range(10)]
+
+data = env.step_batch(states=states, actions=actions)
+new_states, observs, rewards, ends, infos = data
+```
+
+`plangym` allows applying multiple actions in a single call via the command `plangym.Env.step_batch`.
+The syntax used for this case is reminiscent to that employed when calling a `step` function; we should define 
+a __list__ of states and actions and use them as arguments of the function `step_batch()`. `plangym` will
+take care of distributing the states and actions correspondingly, returning a tuple with the results
+of such actions. 
+
+### Making environments
+
+To initialize an environment, `plangym` uses the same syntax as `gym` via the `plangym.make` command. However, 
+this command offers more advanced options than the `gym` standard; it controls the general behavior of the API and
+its different environments, and it serves as a command center between the user and the library. 
+
+Instead of using a specific syntax for each environment (with distinct arguments and parameters),
+`plangym` unifies all options within a single, common framework under the control of
+`make()` command.
+
+All instance attributes are defined through the `make()` command, which classifies and distributes them accordingly whether
+they belong to `plangym` or standard parameters. In addition, `make()` also allows the user to configure the
+parameters needed for stepping the environment in parallel. One only should select the desired mode, and
+`plangym` will do the rest.
+
+```python
+import plangym
+env = plangym.make(
+    name="PlanMontezuma-v0",   # name of the environment
+    n_workers=4,  # Number of parallel processes
+    state='',  # Define a specific state for the environment
+)
+```
+Once the parameters have been introduced, the command instantiates the appropriate environment class
+with the given attributes.
+
+
+#### Make arguments
+
+`make()` accepts multiple arguments when creating an environment. We should distinguish between the arguments
+passed to configurate the environment making process and those used to instantiate the environment itself. 
+* Make signature:  
+Attributes used to configure the process that creates the environment.
+  * `name`:
+  * `n_workers`:
+  * `ray`:
+  * `domain_name`:
```

Comment:
```suggestion
```
`domain-name` is a keyword argument that is only valid for `dm_control` environments, similarly to how `state` is only valid for `RetroEnv`.